assets/icons-white/README.md

```diff
@@ -0,0 +1 @@
+Icons on this folder were found in https://fontawesome.com/
```

Comment:
Please add a link to the license (https://fontawesome.com/license/free) and just say it's CC BY 4.0 directly in the file. Makes it a bit easier.